src/ioctls/vcpu.rs

```diff
@@ -1416,6 +1416,12 @@ impl VcpuFd {
         }
     }
 
+    /// Returns a mutable reference to the kvm_run structure
+    #[allow(clippy::mut_from_ref)]
+    pub fn get_kvm_run(&self) -> &mut kvm_run {
```

Comment:
KvmRunWrapper uses the same pattern unfortunately. This clippy lint is legit. Minting mut refs from const refs is unsafe because you can create aliased mutable refs.

Reply:
I have to admit, it'd be kind of bad to force this at the crate level. Better to make sure the crate doesn't allow for this kind of unsafe behavior.

Reply:
> Should we mark it as unsafe? Seems it breaks the borrow checker.

How did you see it was breaking the borrow checker?

Reply:
We could get multiple mutable reference to the underlying `kvm_run` object:)

Reply:
But now that I updated the code to:
```rust
     pub fn get_kvm_run(&mut self) -> &mut kvm_run { 
```
the borrow checker should be able to do its' job without having to specify `unsafe`, right?

Reply:
I think so:)

Reply:
Ok cool well then the PR is ready!